### DISCUSSION TITLE
How to use React-native-firebase/ml

### DISCUSSION THREAD

**Discussion by adar2378** [invertase/react-native-firebase#5857] [Q&A] [answered]
Hi, I am trying to implement image labeling from image URLs with firebase. I could not find any examples of how to use the React-native-firebase/ml package. Can anyone give me some examples of how to use it? Thanks.

**Comment by mikehardy** [comment, marked as answer]
Hi there! ML was removed upstream in firebase SDKs, so we have removed it here, thus there is no documentation here any longer.
There is more discussion here - #4555 - including (since it's open source...) a way to access our version history to pull up old versions of the docs where ML was included
There are still plans to have a separate (non-firebase) MLKit wrapper for react-native but it has a much smaller audience than firebase (our first priority) and google-ads (also removed from firebase, our second priority at the moment - in progress) so it has taken a while and will continue to take a while. In the meantime I think using the older versions is the best of a set of poor choices - that's unfortunate but is my best thought at the moment.

Good luck with your project

**Reply by adar2378** [reply]
Thank you for the information.
